src/Extensions/TimedScopes/ServiceCollectionExtensions.cs

```diff
@@ -19,6 +20,9 @@ public static class ServiceCollectionExtensions
 		public static IServiceCollection AddTimedScopes(this IServiceCollection serviceCollection)
 		{
 			Activity.DefaultIdFormat = ActivityIdFormat.W3C;
+			serviceCollection.AddHostedService<ActivityObserversIntializer>();
+			serviceCollection.TryAddEnumerable(ServiceDescriptor.Transient<IActivityStopObserver, ActivityStopObserver>());
+
 			serviceCollection.TryAddTransient<IActivityProvider, SimpleActivityProvider>();
```

Comment:
Is there a reason for the line break? In my mind, it's indicating some difference but I'm not seeing one here

Reply:
I've added it split enumerable registration (and hosted that is also enumerable) from Transient, but I don't have a strong opinion and okay removing it if you insisting.

Reply:
It might be good to add a comment clarifying this for future readers

---
In reply to: [394724148](https://github.com/microsoft/Omex/pull/169#discussion_r394724148) [](ancestors = 394724148)

Reply:
No sure that it worth a comment, it's like splitting different blocks in method, if blank line creates confusion we can just remove it